infra/Pos/Infra/Constants.hs

```diff
@@ -37,8 +37,9 @@ data InfraConstants = InfraConstants
     -- ^ We consider node as known if it was pinged at most @ccEnhancedMessageTimeout@ sec ago
     , ccEnhancedMessageBroadcast :: !Word
       -- ^ Number of nodes from batch for enhanced bessage broadcast
-    , ccNetworkReceiveTimeout    :: !Int
-      -- ^ Network timeout on `recv` in milliseconds
+    , ccNetworkWaitLogInterval   :: !Int
+      -- ^ Network wait logging interval in seconds
+      --   (logging that some recv/send takes significant amount of time)
```

Comment:
I don't understand this comment. Can you rephrase it?

Does it mean that if an action takes more `ccNetworkWaitLogInterval` seconds, there'll simply be an entry in the log about it? Or are there other consequences as well?